helm/values.yaml

```diff
@@ -13,9 +13,13 @@ image:
 imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
-volume:
-  class: ""
-  size: 5Gi
+
+persistence:
```

Comment:
This would change the way persistent volumes are declared, could potentially break existing installations when upgrading helm version. Can we add some compatibility mode here ? e.g. if the `volume.size` is present use the old definition rather than the new one?